.github/workflows/ci.yaml

```diff
@@ -69,14 +69,12 @@ jobs:
       - name: "Run tests (Ubuntu)"
         if: ${{ matrix.os == 'ubuntu-latest' }}
         run: |
-          cargo insta test --all --delete-unreferenced-snapshots
```

Comment:
Is the downside here that we no longer flag on unreferenced snapshots? (This used to fail if you included an unused snapshot.)

Reply:
(To be clear, I actually agree with changing this since `cargo insta test` has non-ideal behavior. It'd be nice to have some solution for unreferenced snapshots, but worst-case we can just run this manually from time-to-time.)

Reply:
Yes that's indeed the downside. I think ideally there would be a `cargo insta delete-stray-snapshots` command ... I'll ask Armin about it.

Reply:
In the meantime, is your preference still to merge as-is?

Reply:
I have dropped the first commit implementing this switch to `cargo test` from this PR ... so we can come back to it once there is a solution without any drawbacks :)

Reply:
Just filed https://github.com/mitsuhiko/insta/issues/344 :)

Reply:
This was originally my suggestion so mea culpa. 

But `cargo insta test` definitely used to show diffs, and that is the intended behavior. And looks like it'll be back shortly....

Also FYI there is now `--unreferenced=auto` to avoid the subsequent `git diff`; e.g. https://github.com/PRQL/prql/blob/5de452377e13550d70d7dd5a4d8472a764d85475/.github/workflows/test-rust.yaml#L61